skl2onnx/operator_converters/gaussian_mixture.py

```diff
@@ -0,0 +1,146 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import numpy as np
+from sklearn.mixture.gaussian_mixture import _compute_log_det_cholesky
+from ..common._registration import register_converter
+from ..algebra.onnx_ops import (
+    OnnxAdd, OnnxSub, OnnxMul, OnnxGemm, OnnxReduceSumSquare,
+    OnnxReduceLogSumExp, OnnxExp, OnnxArgMax, OnnxConcat
+)
+
+
+def convert_sklearn_gaussian_mixture(scope, operator, container):
+    """
+    Converter for GaussianMixture.
+
+    Parameters which change the prediction function:
+
+    * *covariance_type*
+    """
+    X = operator.inputs[0]
+    out = operator.outputs
+    op = operator.raw_operator
+    n_features = X.type.shape[1]
+    n_components = op.means_.shape[0]
+
+    # def _estimate_weighted_log_prob(self, X):
```

Comment:
Why commented code?

Reply:
To remember where I found the implementation in scikit-learn.

Reply:
I think it would be better to have comments instead. That would make it clear to anyone reading the code.

Reply:
I think I did (line 29). Does it need more?